hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/client/HddsClientUtils.java

```diff
@@ -48,6 +49,10 @@
 @InterfaceAudience.Public
 @InterfaceStability.Unstable
 public final class HddsClientUtils {
+  @VisibleForTesting
+  static final int MAX_BUCKET_NAME_LENGTH_IN_LOG =
```

Comment:
```suggestion
  static final int MAX_BUCKET_NAME_LENGTH_IN_LOG =
```

I think we can omit the annotation.

Reply:
Makes sense that way too (especially if we are going to keep using the package for utilities), no strong opinion either way.